src/cookbooks/main_ping_exponential_histograms.md

```diff
@@ -32,7 +32,7 @@ As of this writing, each main ping histogram is encoded as a JSON string inside
 SELECT
   payload.histograms.FX_TAB_SWITCH_SPINNER_VISIBLE_MS AS histogram_json,
 FROM
-  telemetry.main
+  telemetry.main_nightly -- Use telemetry.main_1pct for a 1% sample of non-nightly data
```

Comment:
```suggestion
  telemetry.main_nightly -- Use telemetry.main_1pct for a 1% sample across channels
```

Reply:
And we should probably change the `sample_id` value to `0` to be compatible with `main_1pct`. If a user tries to query `main_1pct` with the existing filters here, they will get an empty result set.

Reply:
I think we should at least introduce the concept of `main_nightly` if we're going to use it in the example. A quick sentence above (with a link to more information, if available?) should be fine.